build/WFLY-19130_publish_Wildfly_channel_manifest.adoc

```diff
@@ -0,0 +1,195 @@
+
+A manifest published in an an artifact repository, can be referenced using one of two options - either using a full Maven GAV, or as a stream of manifests,
+using just the GA part of the Maven coordinates.
+When using a full GAV of a manifest, the channel will always retrieve artifact versions associated with this version. On the other hand, when using only a GA,
+the channel will try to resolve the latest available version of the manifest, therefore providing the latest available components. The later scenario is used
+by provisioning tools like Prospero to provide updates for provisioned servers.
+
+If the channel is required to provide backwards compatibility guarantee, this should be enforced using either groupId or artifactId naming convention. 
+For example, if a project provides two concurrent versions (e.g. 1.2.x and 2.0.x), and wants to provide independent updates to both, it could create two
+separate channels `org.test:project-1.2` and `org.test:project-2.0`.
+
+On the other hand, if the channel wants to simply provide access to the latest project state, it can create a single manifest stream with coordinates like
+`org.test:project`. In this case, the newly published manifest might contain a non-backwards compatible upgrade (e.g. from version `1.2` to `2.0`).
+
+The actual versions that the manifest is published with are not relevant, as long as they are incremental. However, if the manifest is tied to a project 
+release (i.e. it can only be updated by releasing a new version of the project), using the same versions as the project will make it easier for the
+users to find correct manifest versions.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-19130[WFLY-19130]
+
+=== Related Issues
+
+
+
+=== Stability Level
+// Choose the planned stability level for the proposed functionality
+* [ ] Experimental
+
+* [ ] Preview
+
+* [X] Community
+
+* [ ] default
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+* mailto:{brian.stansberry@redhat.com}[Brian Stansberry]
+* mailto:{jperkins@redhat.com}[James R. Perkins]
+
+=== QE Contacts
+
+* N/A
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [X] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+* https://github.com/wildfly/wildfly[WildFly]
+* https://github.com/wildfly-extras/prospero[Prospero]
+
+=== Other Interested Projects
+N/A
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
+
+* [x] OpenShift s2i
+
+* [x] Bootable jar
+
+== Requirements
+
+WildFly project currently creates three feature packs: `wildfly-ee-galleon-pack`, `wildfly-galleon-pack` and `preview-galleon-pack`.
```

Comment:
These are 'wildfly-ee', 'wildfly' and 'wildfly-preview'. The unfortunately varied artifact ids are not how we refer to them in documentation unless we are specifically documenting artifactIds. (Also, the artifactId of wildfly-preview is 'wildfly-preview-feature-pack'.)